packages/nuxt/src/app/plugins/chunk-reload.client.ts

```diff
@@ -0,0 +1,23 @@
+import { defineNuxtPlugin } from '#app/nuxt'
+import { useRouter } from '#app/composables/router'
+
+export default defineNuxtPlugin((nuxtApp) => {
+  const router = useRouter()
+
+  let chunkError: TypeError | null = null
+  nuxtApp.hook('app:chunkError', ({ error }) => { chunkError = error })
+
+  router.onError((error, to) => {
+    if (chunkError !== error) { return }
+
+    let handledPath: Record<string, any> = {}
+    try {
+      handledPath = JSON.parse(localStorage.getItem('n:chunk:reload') ?? '{}')
```

Comment:
I would suggest to rename it to `nuxt` for readability since is not transformed over network key size should be okay.. (also empty string cannot be parsed)

```suggestion
      handledPath = JSON.parse(localStorage.getItem('nuxt:reload') || '{}')
```